app/hooks/useSettingsAction.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { createAction } from "~/actions";
+import history from "~/utils/history";
+import useAuthorizedSettingsConfig from "./useAuthorizedSettingsConfig";
+
+const useSettingsActions = () => {
+  const config = useAuthorizedSettingsConfig();
+  const actions = React.useMemo(() => {
+    return config.map((item) => {
+      const Icon = item.icon;
+      return createAction({
```

Comment:
I'd say we want these to be children of the "Settings" action, because it's a bit confusing in your screenshot. With the parent it would read "Settings -> Features" for example, and child actions are still searchable

Reply:
What would be the group of the parent "Settings" action?

Reply:
I think "Navigation" would be fine